topics/gradescope/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: Gradescope
+topic: gradescope
+github_url: https://github.com/gradescope
+logo: gradescope.png
+related: autograder
+released: 2014
+short_description: Gradescope is a suite of services to make grading coursework and providing feedback easier.
+url: https://www.gradescope.com/
+---
+Gradescope is a suite of services to make grading coursework and providing feedback easier. Gradescope streamlines the tedious parts of grading paper-based, digital, and code assignments while providing insights into how your students are doing. Gradescope lets instructors save time on grading so that they can do more of what matters – teaching! With Gradescope, students receive faster and more detailed feedback on their work, and instructors can see detailed assignment and question analytics.
```

Comment:
Grammar Fix

```suggestion
Gradescope is a suite of services to make grading coursework and providing feedback easier. Gradescope streamlines the tedious parts of grading paper-based, digital, and code assignments while providing insights into how your students are doing. Gradescope lets instructors save time on grading so that they can do more of what matters – teaching! With Gradescope, students receive faster and more detailed feedback on their work, and instructors can see the detailed assignment and question analytics.
```